sos/report/plugins/foreman_installer.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (C) 2021 Red Hat, Inc., Pavel Moravec <pmoravec@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import (Plugin, RedHatPlugin, DebianPlugin,
+                                UbuntuPlugin)
+
+
+class ForemanInstaller(Plugin, DebianPlugin, UbuntuPlugin):
+
+    short_desc = 'Foreman installer and maintainer'
+
+    plugin_name = 'foreman_installer'
+    profiles = ('sysmgmt',)
+    packages = ('foreman-installer', 'rubygem-foreman_maintain')
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/foreman-installer/",
```

Comment:
Why does this line not have `/*`? Is that not needed? Something I recently noticed is that backups also include `/etc/foreman-installer/scenarios.d/*.migrations/*.rb` but those are shipped by the RPM. I think it's highly unlikely that it's ever needed. Something you may want to consider. Note that `/etc/foreman-installer/scenarios.d/*.migrations` are actually symlinks to `/usr/share/foreman-installer`.

Reply:
I recall there were some issues where `add_copy_spec` really needs '/*' for dirs, but I dont recall why. Just to be sure, I will add the asterisk there.

To the `*rb` files: indeed, these files shall be redundant. The only harm from them is they occupy some small space, but.. why not forbid collecting them.

The `.applied` symlinks collection: indeed, that will be a replacement of the `*rb` files (with particular info what script was applied). We didnt collect it until now as we dont collect hidden files until explicitly mentioned. Adding it now..

To the "Satellite 6" - good point, removing the six.